### PR TITLE
MMT-1218 Collection Data Type Display

### PR DESCRIPTION
**Pull request by zjam1**
When on the collection preview page. If the Collection Data Type is Near Real Time then the NRT badge displays next to the subtitle in the header section.

### CODE REVIEW

**Comment by jceaser** [comment]
need unit test to test for the existence of the tag

File: app/views/collection_drafts/show.html.erb [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <% content_for :header_title do %>
   <h2><%= display_entry_id(get_resource.draft, resource_name) %></h2>
   <p class="subtitle"><%= display_collection_entry_title(get_resource.draft) %></p>
+  <% if get_resource.draft['CollectionDataType'] == 'NEAR_REAL_TIME' %>
```
Comment (jceaser) [inline app/views/collection_drafts/show.html.erb]: these next three sections look like they mostly repeat, maybe a helper function

File: spec/features/collection_drafts/breadcrumbs_and_header_spec.rb [hunk 2]
```diff
@@ -31,7 +31,7 @@
       visit collection_draft_path(@draft)
     end
 
-    it 'displays the short name' do
+    it 'displays the short name within in breadcrumbs' do
```
Comment (charleshuang80) [inline spec/features/collection_drafts/breadcrumbs_and_header_spec.rb]: really minor, but I think there's a typo. the 'in' after 'within' should be changed or removed.

File: app/helpers/collections_helper.rb [hunk 3]
```diff
@@ -34,4 +34,10 @@ def render_collection_download_xml_link(download_format, download_link_title, co
     link_to(download_link_title, download_collection_xml_path(concept_id, download_format, revision_id),
             id: download_format)
   end
+
+  def nrt_badge(metadata)
+    if metadata['CollectionDataType'] == 'NEAR_REAL_TIME'
+      content_tag :span, 'NRT', class: 'eui-badge nrt'
```
Comment (charleshuang80) [inline app/helpers/collections_helper.rb]: I know parentheses aren't required here, but if you look at where we use `content_tag` in the rest of the project, we wrap the arguments in parentheses, so it we should be consistent with that.

File: app/views/collection_drafts/show.html.erb [hunk 4]
```diff
@@ -1,6 +1,7 @@
 <% content_for :header_title do %>
   <h2><%= display_entry_id(get_resource.draft, resource_name) %></h2>
   <p class="subtitle"><%= display_collection_entry_title(get_resource.draft) %></p>
+  <%= nrt_badge get_resource.draft %>
```
Comment (charleshuang80) [inline app/views/collection_drafts/show.html.erb]: also please use parentheses with this helper to be consistent. as you can see, the view helpers above are called with parentheses.

File: app/views/collections/revisions.html.erb [hunk 5]
```diff
@@ -1,6 +1,7 @@
 <% content_for :header_title do %>
   <h2><%= display_entry_id(@collection, 'collections') %></h2>
   <p class="subtitle"><%= display_collection_entry_title(@collection) %></p>
+  <%= nrt_badge @collection %>
```
Comment (charleshuang80) [inline app/views/collections/revisions.html.erb]: and here

File: app/views/collections/show.html.erb [hunk 6]
```diff
@@ -1,6 +1,7 @@
 <% content_for :header_title do %>
   <h2><%= display_entry_id(@collection, 'collections') %></h2>
   <p class="subtitle"><%= display_collection_entry_title(@collection) %></p>
+  <%= nrt_badge @collection %>
```
Comment (charleshuang80) [inline app/views/collections/show.html.erb]: and here

**Pull request merged**